### DISCUSSION TITLE
custom increments

### DISCUSSION THREAD

**Discussion by Darecki626** [Araxeus/Youtube-Volume-Scroll#7] [Q&A] [answered]
Is it possible to add ability to set custom intervals between volume levels, for example increments of 5 10 etc. 1 is to "slow"

**Comment by Araxeus** [comment, marked as answer]
It is already possible. click the extension icon to access it 😀

![image](https://user-images.githubusercontent.com/78568641/152661718-d1b65faf-81e5-4cf5-b741-b48e68518b88.png)

**Comment by Darecki626** [comment]
me dumb 😐 😃 well expected it to be in options as in most of the extension, and this one not having one :P

**Reply by Araxeus** [reply]
I made it like this because its supposed to be easier to access and I think it looker nicer too
(and you can also open this menu while browsing youtube)
